tests/synthdnn/README.md

```diff
@@ -0,0 +1,28 @@
+# Synthdnn
+
+Synthdnn is a suite of scripts for collecting and analyzing oneDNN performance
+across a randomly generated data. The general architecture is intended to follow
+a data pipeline composed of synthetic problem generation, data collection, and
+data analysis. The `synthdnn.py` script provides a command line interface to
+these tools. Sample Usage:
+
+
+Data Generation:
```

Comment:
Suggest to differentiate and rename:
`Data` that refers to benchdnn problems or cases to be renamed into `Problems` or `Cases`.
`Data` that refers to performance numbers to be renamed into `Performance Data`.
Otherwise, the context of different data might be mixed up.